qiita_pet/handlers/analysis_handlers.py

```diff
@@ -141,14 +142,17 @@ def get(self, analysis_id):
             jobres[jobject.datatype].append((jobject.command[0],
                                              jobject.results))
 
-        dropped = {}
         dropped_samples = analysis.dropped_samples
+        dropped = defaultdict(list)
         if dropped_samples:
-            for proc_data_id, samples in viewitems(dropped_samples):
+            for proc_data_id, samples in viewitems(analysis.dropped_samples):
```

Comment:
Why did you change this line? `dropped_samples` was already set to `analysis.dropped_samples` on line 145